docs/install/knative-with-operators.md

```diff
@@ -40,19 +40,19 @@ Information about Knative Serving Operator releases can be found on the [Release
 
 __From releases__:
 
-Replace \<version\> with the latest version or the version you would like to install, and run the following command to
-install Knative Serving Operator:
+Run the following command to install Knative Serving Operator:
 
 ```
-kubectl apply -f https://github.com/knative/serving-operator/releases/download/<version>/serving-operator.yaml
+kubectl apply --filename {{< artifact repo="serving-operator" file="serving-operator.yaml" >}}
 ```
 
 __From source code__:
 
 You can also install Knative Operators from source using `ko`.
 
 1. Install the [ko]((https://github.com/google/ko)) build tool.
```

Comment:
This is another error I brought in with the  double brackets.
It should be [ko](https://github.com/google/ko)